pkg/registry/core/node/storage/storage_test.go

```diff
@@ -69,22 +71,47 @@ func validNewNode() *api.Node {
 			},
 		},
 	}
+
+	for _, tweak := range tweaks {
+		tweak(node)
+	}
+
+	return node
+}
+
+func setNodeIPAddress(addr string) tweak {
+	return func(node *api.Node) {
```

Comment:
This will un-do a call to `setNodeDaemonEndpoint` because it totally over-writes the status.

Ideally these are all composable and only allocat a struct if it is not already present

Reply:
Should be fixed now.